build/Dockerfile.initcontainer

```diff
@@ -4,7 +4,7 @@ RUN apk add --update \
     bc \
     build-base \
     curl \
-    libelf-dev \
+    elfutils-dev \
```

Comment:
what is different in these two packages? no big deal, just curious

Reply:
libelf-dev is not available in newer alpine versions and best I can tell was based on https://github.com/WolfgangSt/libelf (which is a mirror?). elfutils and elfutils-dev seem to be the more standard packages and at least the header setup steps for the Amazon Linux kernel version wouldn't work with libelf-dev. Curious to hear if this causes any trouble with newer kernel versions.

Reply:
i think it is just a requirement for generating the headers, it is likely fine? Assuming you've done a test run w/ it

Reply:
Yeah, it worked for the Amazon Linux kernel version at least. If you're able to give it a run with a more modern GCP kernel that would be great.